spectrum-offchain-lm/src/data/bundle.rs

```diff
@@ -140,24 +183,18 @@ impl IntoBoxCandidate for StakingBundle {
 impl TryFromBox for StakingBundle {
     fn try_from_box(bx: ErgoBox) -> Option<StakingBundle> {
         if let Some(ref tokens) = bx.tokens {
-            if tokens.len() == 2 && bx.ergo_tree == bundle_validator() {
+            if tokens.len() == 3 && bx.ergo_tree == bundle_validator() {
```

Comment:
Reflecting new changes in https://github.com/spectrum-finance/ergo-dex/blob/67a4d8d4dfb1522131a233befb37da805cba8299/contracts/lqmining/simple/StakingBundle.sc